radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -393,7 +393,7 @@ public TokenTypeReference getNativeTokenRef() {
 		return this.getActionStore(actionClass).getActions(address, identity)
 			.map(actionClass::cast)
 			.publish()
-			.autoConnect()
+			.refCount()
```

Comment:
Just a self-educational question: the difference here is that `refCount()` also disconnects the underlying observable when the last observer is detached, whereas the `autoConnect()` only guarantees that the underlying observable is connected (but not disconnected), right?

Reply:
Yes, exactly. And after last observer is detached and then a new observer subscribes, `refCount` will start a new flow while `autoConnect` will still be connected to the original.